README.md

```diff
@@ -1047,7 +1047,7 @@ chaining methods like `map`, `slice`, `forEach` etc. See [itiriri](https://www.n
 an example of advanced manipulation with iterables (or [itiriri-async](https://www.npmjs.com/package/itiriri-async) for manipulation of async iterables).
 
 ```ts
-import itiriri from 'itiriri';
```

Comment:
`itiriri` is currently under development of v2, where:
 
`import { query } from 'itiriri'` 

is changed to default import:

`import itiriri from 'itiriri'`. 

V2 is planned to be published in the upcoming week. 
I'd suggest that we leave it as it was in order to avoid a new commit to update this a week later.